integration_tests/test_use_walrus_if.py

```diff
@@ -0,0 +1,32 @@
+from codemodder.codemods.use_walrus_if import UseWalrusIf
+from integration_tests.base_test import (
+    BaseIntegrationTest,
+    original_and_expected_from_code_path,
+)
+
+
+class TestUseWalrusIf(BaseIntegrationTest):
```

Comment:
@andrecsilva the integration test is here. Is there something specific that is missing?

Reply:
Nah, I've somehow missed it. Sorry